api/register/swagger-api_register.yaml

```diff
@@ -39,6 +39,36 @@ paths:
           description: Invalid information provided
         403:
           description: Invalid RECAPTCHA token
+  /register/realms/{realm}/user:
+    post:
+      tags:
+      - Register
+      summary: Creates a user
+      security:
+        - BasicAuth: [recaptcha]
+      parameters:
+      - name: realm
+        in: path
+        description: realm name (not id!)
```

Comment:
When we create a user in a specific realm, I don't see a scenario where we would need to use the realm configuration of another realm so I would suggest to remove this parameter. We would always use the realm provided in URL

Reply:
Here, realm is a path parameter. In first version, realm was a query parameter.

Reply:
ohh yes, sorry for it